mlrun/api/db/sqldb/db.py

```diff
@@ -2104,7 +2104,9 @@ def store_feature_set(
         if uid == existing_feature_set.uid or always_overwrite:
             db_feature_set = existing_feature_set
         else:
-            db_feature_set = FeatureSet(project=project, full_object=feature_set_dict)
+            db_feature_set = FeatureSet(
+                project=project, created=feature_set_dict["metadata"].get("created")
+            )
```

Comment:
I'm a little confused - does this change fix anything? The fset fields are later handled in the `_update_db_record_from_object_dict` method, including filling the `created` field - I do agree that passing the `full_object` here is redundant, but so is the `created` field, so why did you do this change?

Reply:
a few points to understand here:
- `full_object` has a setter that does `json.load` on the `feature_set_dict` which fails because the dict has `datetime` objects in it (`created` and `updated`) which are not JSON serielizable.
- the reason that setting `full_object` in `_update_db_record_from_object_dict` works is because the datetime fields are being handled (notice they are casted to `str`)
```
common_object_dict["metadata"]["updated"] = str(updated_datetime)
common_object_dict["metadata"]["created"] = str(db_object.created)
```
-  the `created` field in `_update_db_record_from_object_dict` is being taken from the db object (see code block from point (2)) and not from the dict. if it will be empty, a new creation time will be set.

Reply:
👍 
Understood on the `full_object` - as discussed, just change the `created` and that's it.